doc/merging.rst

```diff
@@ -0,0 +1,56 @@
+How to merge a branch after approving the PR
+============================================
+
+Requirements
+--------------
+
+- A remote called *upstream* pointing at the central IC repository.
```

Comment:
It is not a *requirement* that the remote be called `upstream` (you can call it whatever you like), but I'm reluctant to add this detail, as it would significantly lengthen a beautifully short line, and requiring it will probably help people from confusing themselves. So best leave it as it is.

`upstream` should be rendered in `code-face` (can't remember how it's done in RST), but the stars render it in *emphasis-face*. Same fix needed in lots more places throughout the document below.